### ISSUE TITLE
Would you like to give us an example to use swa callback?

### ISSUE THREAD

**Issue by xuzhang5788** [kristpapadopoulos/keras_callbacks#2]
Thank you so much for your code. 
It will be great if you would like to upload an example to show how to use your callback. If I use learning_rate_schedular, may I combine your saw together? Because you said that your learning rate is a constant. Many thanks.

**Comment by kristpapadopoulos** [comment]
In the readme there is a simple example on how to use it.  it was only designed with a constant learning rate, for a variable learning rate a different implementation would be required.
